spec/factories/service.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :service do
+    digit = Faker::Number.between(1, 10)
+
+    code { Faker::Number.number(digit) }
+    name { Faker::Name.name }
+    description { Faker::Lorem.paragraph }
+    value { Faker::Number.decimal(digit, 2) }
```

Comment:
It's not neccessary to do this, you don't need line 3.
On line 8 you can just do Faker::Number.positive.round(2)